discovery-provider/compose/docker-compose.common.yml

```diff
@@ -0,0 +1,19 @@
+version: '3'
+services:
+  discovery-provider:
+    build: ../.
+    restart: always
+    env_file:
+      - .env
+    environment:
+      - FLASK_RUN_PORT=${audius_server_port}
+      - audius_discprov_dev_mode=true
+      - audius_ipfs_host=${COMPOSE_PROJECT_NAME}-ipfs-node
+      - audius_redis_url=redis://${COMPOSE_PROJECT_NAME}_redis-server_1:6379/00
+      - audius_delegate_owner_wallet=${audius_delegate_owner_wallet}
+      - audius_delegate_private_key=${audius_delegate_private_key}
```

Comment:
I need to confirm that `audius_delegate_*` envars shouldn't cause any issues when set within `docker-compose.web-server.yml`.